04/styles/global.css

```diff
@@ -0,0 +1,67 @@
+/* ./styles/global.css */
+
+:root {
+    --color-alfa: rgba(255, 255, 255, 1);
+    --color-beta: rgba(0, 0, 0, 1);
+    --color-gamma: rgba(71, 175, 245, 1);
+    --color-omega: rgba(36, 41, 46, 1);
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.nav {
+	padding: 0;
+	margin: 0;
+    list-style-type: none;
+	font-size: 24px;
+    background-color: var(--color-alfa);
+}
+
+a {
+    color: var(--color-gamma);
+	text-decoration: none;
+    display: block;
+}
+
+/* Hamburger */
+/* .hamburger {
```

Comment:
Trzeba uważać z wysyłaniem zakomentowanego kodu na githuba.
Inna osoba (np. ja) nie wiem czy to jest do sprawdzenia czy do późniejszego wykorzystania - czy jak?! :)